client/my-sites/hosting/hosting-activate.js

```diff
@@ -11,8 +11,8 @@ import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selecto
 const HostingActivate = ( { initiateTransfer, siteId, siteSlug, translate } ) => {
 	const backUrl = `/hosting-config/${ siteSlug }`;
 
-	const transferInitiate = () => {
-		initiateTransfer( siteId, null, null );
+	const transferInitiate = ( { geo_affinity = null } ) => {
```

Comment:
Everywhere else `geo_affinity` is defaulted it uses `''` to ensure the value is always a string. Personally I like using `null` to represent empty 😄 but why break the convention for this case?

Reply:
> Everywhere else `geo_affinity` is defaulted it uses `''` to ensure the value is always a string. Personally I like using `null` to represent empty 😄 but why break the convention for this case?

Good point. Updated with 3ffa6e64b4a257210f64c37dfda57d00fe99a559